ui/src/Panels/Consumer/Consumer.assets.js

```diff
@@ -0,0 +1,14 @@
+import translations from './Consumer.i18n.json';
+import { CONSTANTS } from 'Utils';
+import {
+  storybookWebsocket,
+  controlledWebsocket,
+} from 'DevUtils/MockWebsocket';
+
+export { translations };
+
+export const consumerMockWebsocket = () =>
+  storybookWebsocket(CONSTANTS.CONSUMER);
+
+export const consumerMockWebsocketForTest = () =>
+  controlledWebsocket(CONSTANTS.CONSUMER);
```

Comment:
This file has assets for testing alongside assets needed in production - would it be worth splitting them out? In PR #100 I created a `Messages.spec.assets.js` file for use by both the `spec.js` and `stories.js` files. A production build could then tree-shake these files out.

Reply:
We should be good here - we have set up the build to treeshake at the import/export level - so constant by constant, function by function, only what is used in prod will go out in the bundle